e2e-tests/gef/cypress/e2e/facility/about-facility.spec.js

```diff
@@ -50,6 +50,10 @@ context('About Facility Page', () => {
       aboutFacility.coverEndDateDay();
       aboutFacility.coverEndDateMonth();
       aboutFacility.coverEndDateYear();
+      if (applications[1].version >= 1) {
```

Comment:
This will need feature flagging one way or another, so I figured just looking at the deal directly is the best way?
I could get the `GEF_DEAL_VERSION` as an env var and do `if(GEF_DEAL_VERSION >=1)`?